pkg/scanner/local/scan.go

```diff
@@ -300,26 +305,32 @@ func (s Scanner) scanLicenses(target types.ScanTarget, options types.ScanOptions
 	}
 
 	// License - file header or license file
-	var fileLicenses []types.DetectedLicense
-	for _, license := range target.Licenses {
-		for _, finding := range license.Findings {
-			category, severity := scanner.Scan(finding.Name)
-			fileLicenses = append(fileLicenses, types.DetectedLicense{
-				Severity:   severity,
-				Category:   category,
-				FilePath:   license.FilePath,
-				Name:       finding.Name,
-				Confidence: finding.Confidence,
-				Link:       finding.Link,
-			})
+	if options.LicenseFull {
```

Comment:
Similar case.

We only keep file licenses and dpkg licenses (but we remove them in `applier`) in this slice.

So the report can only include file licenses.
We don't need to show this result if the `--liecense-full` flag is not set.